File: SemantikosClientesWS/ClienteJava/src/cl/minsal/semantikos/ws/shared/Categoria.java

package cl.minsal.semantikos.ws.shared;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlType;


/**
 * <p>Java class for Categoria complex type.
 * 
 * <p>The following schema fragment specifies the expected content contained within this class.
 * 
 * <pre>
 * &lt;complexType name="Categoria">
 *   &lt;complexContent>
 *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
 *       &lt;sequence>
 *         &lt;element name="nombre" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="nombreAbreviado" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="restringida" type="{http://www.w3.org/2001/XMLSchema}boolean" minOccurs="0"/>
 *       &lt;/sequence>
 *     &lt;/restriction>
 *   &lt;/complexContent>
 * &lt;/complexType>
 * </pre>
 * 
 * 
 */
@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "Categoria", propOrder = {
    "nombre",
    "nombreAbreviado",
    "restringida"
})
public class Categoria {

    protected String nombre;
    protected String nombreAbreviado;
    protected Boolean restringida;

    /**
     * Gets the value of the nombre property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getNombre() {
        return nombre;
    }

    /**
     * Sets the value of the nombre property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setNombre(String value) {
        this.nombre = value;
    }

    /**
     * Gets the value of the nombreAbreviado property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getNombreAbreviado() {
        return nombreAbreviado;
    }

    /**
     * Sets the value of the nombreAbreviado property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setNombreAbreviado(String value) {
        this.nombreAbreviado = value;
    }

    /**
     * Gets the value of the restringida property.
     * 
     * @return
     *     possible object is
     *     {@link Boolean }
     *     
     */
    public Boolean isRestringida() {
        return restringida;
    }

    /**
     * Sets the value of the restringida property.
     * 
     * @param value
     *     allowed object is
     *     {@link Boolean }
     *     
     */
    public void setRestringida(Boolean value) {
        this.restringida = value;
    }

}

File: SemantikosClientesWS/clientePHP/src/Semantikos/ClientBundle/API/Categoria.php
<?php

namespace Semantikos\ClientBundle\API;

class Categoria
{

    /**
     * @var string $nombre
     */
    protected $nombre = null;

    /**
     * @var string $nombreAbreviado
     */
    protected $nombreAbreviado = null;

    /**
     * @var boolean $restringida
     */
    protected $restringida = null;

    
    public function __construct()
    {
    
    }

    /**
     * @return string
     */
    public function getNombre()
    {
      return $this->nombre;
    }

    /**
     * @param string $nombre
     * @return Categoria
     */
    public function setNombre($nombre)
    {
      $this->nombre = $nombre;
      return $this;
    }

    /**
     * @return string
     */
    public function getNombreAbreviado()
    {
      return $this->nombreAbreviado;
    }

    /**
     * @param string $nombreAbreviado
     * @return Categoria
     */
    public function setNombreAbreviado($nombreAbreviado)
    {
      $this->nombreAbreviado = $nombreAbreviado;
      return $this;
    }

    /**
     * @return boolean
     */
    public function getRestringida()
    {
      return $this->restringida;
    }

    /**
     * @param boolean $restringida
     * @return Categoria
     */
    public function setRestringida($restringida)
    {
      $this->restringida = $restringida;
      return $this;
    }

}
